langworld_db_data/adders/listed_value_adder.py

```diff
@@ -253,13 +259,17 @@ def _increment_value_ids_in_feature_profiles(
                     f"{target_feature_id}{ID_SEPARATOR}{current_value_index + 1}"
                 )
                 row["value_id"] = incremented_current_value_id
+                is_changed = True
+                print(True)
```

Comment:
please make sure you delete `print` statements

Reply:
Done! I checked the other prints, they do not appear to serve for debugging like this one so they remain intact